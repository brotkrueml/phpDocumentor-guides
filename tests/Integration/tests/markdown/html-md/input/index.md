# Markdown with html

<div align="center">
In files of text, where words take flight,
Markdown weaves its magic, bold and bright.
Hashes and stars, a simple code,
A poet's playground, where stories unfold.
</div>