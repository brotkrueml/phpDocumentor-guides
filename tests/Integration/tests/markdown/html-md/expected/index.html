<!DOCTYPE html>
<html lang="en">
<head>
    <title>Markdown with html - Project Title</title>

</head>
<body>
    <h1>Markdown with html</h1>

    <p>In files of text, where words take flight,
    Markdown weaves its magic, bold and bright.
    Hashes and stars, a simple code,
    A poet's playground, where stories unfold.</p>
</body>
</html>
